### PR TITLE
🧖‍♂️ Add delete function

### LINKED ISSUE #128: Delete function
Adjust backend
Add function to frontend

### PR DESCRIPTION
**Pull request by bwnstck**
# Description

Fixes #128 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: client/src/pages/AddTour.js [hunk 1]
```diff
@@ -25,8 +25,9 @@ export default function AddTour() {
     cargo: null,
     carriage: false,
     assignment: "",
+    checkboxes: [],
   });
-  const [checkboxes, setCheckboxes] = useState([]);
+  // const [checkboxes, setCheckboxes] = useState([]);
```
Comment (arassaria) [inline client/src/pages/AddTour.js]: Delete, don't comment

**Review by arassaria — CHANGES_REQUESTED** [review]
Not allowed to merge, till you fix this 😈

**Review by arassaria — APPROVED** [review]
🦄 LGTM 🚀

**Review by phlgr — APPROVED** [review]
LGTM 🚀

**Pull request merged**